test/process_test.cc

```diff
@@ -22,6 +23,37 @@ using namespace testing;
 namespace Nighthawk {
 namespace Client {
 
+constexpr absl::string_view kSinkName = "{name:\"nighthawk.dummy_stats_sink\"}";
```

Comment:
While this might technically be OK and the absl folks seem really happy with it, I wonder what's wrong with `constexpr const char kFoo[] = "";`. Maybe we should discuss this in the context of NH too.
/cc @jmarantz because we had a similar back and fort a little while ago.

Reply:
Changed to `constexpr const char kFoo[] = "";` to follow NH convention

Reply:
Thanks; somehow I find that's easier for the reader to see that this won't cause trouble, as it's just a plain old data type.

Reply:
Ah, after I changed to `constexpr const char kFoo[] = "";`, clang-tidy report error `do not declare C-style arrays, use std::array<> instead [modernize-avoid-c-arrays,-warnings-as-errors]`

So I will change back to `constexpr absl::string_view kSinkName =""` if that is ok

Reply:
Well, if clang-tidy objects, I don't think we have much choice. Looking at this recommendation,
it should be allright: https://abseil.io/tips/168. Sorry about the fuzz.